### PR TITLE
First practice

### PR DESCRIPTION
**Pull request by KaiYang1010**
This contains files and instructions for MD-bootcamp's first simulation practice

### CODE REVIEW

File: MD_bootcamp/1-Hoffman2_Cluster/2-Useful-Linux-Command.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# Basic linux command (for MD-simulation purpose only)
+This page only contains frequently used linux (or other) commands for our MD-simulation.   
+If you want to know more about linux commands,  
+link to **Linux command cheat sheet** is [here](https://files.fosswire.com/2007/08/fwunixref.pdf).
+
+You can watch Prof.Bauchy's recorded video [here](https://www.youtube.com/watch?v=PUW__yVsYSY&ab_channel=PARISlab%40UCLA), and below is my summary.
+
+## General hints (Updating...)
+1. Commands are case sensitive (i.e., ls works, but LS does not word)
```
Comment (YuhaiLi78) [inline MD_bootcamp/1-Hoffman2_Cluster/2-Useful-Linux-Command.md]: typo: word -> work

Comment (KaiYang1010) [reply MD_bootcamp/1-Hoffman2_Cluster/2-Useful-Linux-Command.md]: Got it, thx

File: MD_bootcamp/1-Hoffman2_Cluster/2-Useful-Linux-Command.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+2. Use 'TAB' to save time  
+for example, if you have two directories are named 'abcde' and 'xyz', you just need to type 'a' then press TAB button, your terminal will show 'abcde' automatically
+3. Frequently check your storage since each account only has limited 20 GB space (See commmand below)
+
+
+## File management
+After you login to Hoffman2 Cluster using your local machine, you can
+- List files in your current directory (folder)
+```python
```
Comment (YuhaiLi78) [inline MD_bootcamp/1-Hoffman2_Cluster/2-Useful-Linux-Command.md:18]: why python code block?

File: MD_bootcamp/1-Hoffman2_Cluster/2-Useful-Linux-Command.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+mkdir YourDirName
+```
+
+- Check your current directory
+```
+pwd
+```
+
+- Change directory (most used examples based on my experience, but there are more)
+```python
```
Comment (YuhaiLi78) [inline MD_bootcamp/1-Hoffman2_Cluster/2-Useful-Linux-Command.md:33]: same as above

File: MD_bootcamp/1-Hoffman2_Cluster/2-Useful-Linux-Command.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+cd                  # Move to your account home directory
+cd YourDirName      # enter YourDirName subdirectory 
+cd Dir1/Dir2/Dir3   # enter Dir3 under Dir2, which is also under Dir1
+cd ..               # Change to parent directory
+cd ../Dir4          # Change to parent directory then enter Dir5 directory
+```
+
+- Remove directory
+```
+rmdir YourDirName
```
Comment (YuhaiLi78) [inline MD_bootcamp/1-Hoffman2_Cluster/2-Useful-Linux-Command.md:43]: a more commonly used command is `rm -rf YourDirName`

File: MD_bootcamp/1-Hoffman2_Cluster/3-How-to-submit-job.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+# How to submit a job to Hoffman2 Cluster 
+Finally, you are going to submit your first simulation!!!
+
+There are diffent ways you can submit your job to Hoffman2 Cluster, let's first try the very basic one, and I will show you a better way to do it for next exercise.
+
+## Download input files and upload to Hoffman2
+Please go to ~/MD_bootcamp/5-Practice/1-SiO2 in this GitHub repo, and download all 4 files.
```
Comment (YuhaiLi78) [inline MD_bootcamp/1-Hoffman2_Cluster/3-How-to-submit-job.md]: good to add a link to this folder

**Review by YuhaiLi78 — COMMENTED** [review]
some comments to be considered

File: MD_bootcamp/1-Hoffman2_Cluster/2-Useful-Linux-Command.md (hunk 2 above)
Comment (KaiYang1010) [reply MD_bootcamp/1-Hoffman2_Cluster/2-Useful-Linux-Command.md:18]: 老子愿意

Comment (YuhaiLi78) [reply MD_bootcamp/1-Hoffman2_Cluster/2-Useful-Linux-Command.md:18]: Orz

File: MD_bootcamp/1-Hoffman2_Cluster/3-How-to-submit-job.md (hunk 5 above)
Comment (KaiYang1010) [reply MD_bootcamp/1-Hoffman2_Cluster/3-How-to-submit-job.md]: How to do this before I merge?
Will the directory change after merging?
Can you help me edit this one?

Comment (YuhaiLi78) [reply MD_bootcamp/1-Hoffman2_Cluster/3-How-to-submit-job.md]: you may use a relative path, i.e. ../5-Practice/1-SiO2

**Review by YuhaiLi78 — APPROVED** [review]
Looks good

**Pull request merged**